bitbots_body_behavior/src/bitbots_body_behavior/main.dsd

```diff
@@ -2,25 +2,27 @@
 @ChangeAction + action:searching, @LookAtFieldFeatures, @StandAndWait + duration:4, @TurnAround
 
 #DoNothing
-@ChangeAction + action:waiting, @LookForward, @StandAndWait
+@ChangeAction + action:waiting, @LookForward, @Stop
 
 #StandAndLook
 @ChangeAction + action:waiting, @LookAtFieldFeatures, @StandAndWait
 
 #GetWalkreadyAndLocalize
-@ChangeAction + action:waiting + r:false, @LookForward + r:false, @PlayAnimationInit + r:false, @GetWalkready + r:false, @LookAtFieldFeatures + r:false, @StandAndWait + duration:2 + r:false, @StandAndWait
+@ChangeAction + action:waiting + r:false, @LookForward + r:false, @PlayAnimationInit + r:false, @GetWalkready + r:false, @LookAtFieldFeatures + r:false, @StandAndWait
```

Comment:
```suggestion
@ChangeAction + action:waiting + r:false, @LookForward + r:false, @PlayAnimationInit + r:false, @LookAtFieldFeatures + r:false, @GetWalkready + r:false, @StandAndWait
```
The walkready time is now used for localization, therefore we should look around.